pkg/api/server/v1alpha2/records_test.go

```diff
@@ -255,6 +255,7 @@ func TestListRecords(t *testing.T) {
 	records := make([]*pb.Record, 0, 6)
 	// Create 3 TaskRun records
 	for i := 0; i < 3; i++ {
+		fakeClock.Advance(1)
```

Comment:
Few other instances of `Advance(1)` that we might want to change to `Advance(time.Second)`

Reply:
D'oh, good catch!  All uses of `Advance(1)` should be replaced now.